x-pack/test/security_solution_endpoint/apps/endpoint/policy_details.ts

```diff
@@ -241,7 +241,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
                 popup: {
                   malware: {
                     enabled: true,
-                    message: 'Elastic Security { action } { filename }',
+                    message: 'Elastic Security {action} {filename}',
```

Comment:
You see how fragile these tests are because you have to change this in so many places?

Only 1 test should care about the contents of that message. The test testing the popup malware message. The rest should not care what's in that string, because there is a dedicated test to look at it. So if it changes, only 1 test fails. Not 6.

And then devs won't have to change it in so many places. Just the code, and the 1 test testing that thing.

Reply:
please have a testing class thank you <3